File: loianeTraining/javaBasico/exercicios-aulas-18-19/Ex24.java
class Ex24 {
    public static void main(String[] args) {
        int[] a = { 2, 3, 3, 2, 1 };

        for (int i = 0; i < a.length / 2; i++) {
            if (a[i] != a[a.length - i - 1]) {
                System.out.println("Nao e palindrometro");
                return;
            }
        }
        System.out.println("E um palindrometro");
    }
}

File: loiane-training-csharp/procedural-poo/e/Ex24.cs
using System;

class Ex24
{
    static void Main(string[] args)
    {
        double[] vetorA = new double[10] { 1, 2, 3, 4, 5, 5, 4, 3, 2, 0 };

        bool palindrometro = true;
        for (int i = 0, j = 9; i < vetorA.GetLength(0) / 2; i++, j--)
        {
            if (!(vetorA[i] == vetorA[j]))
            {
                palindrometro = false;
            }
        }

        Console.WriteLine("\nVetor e um Palindrometro? {0}", palindrometro);
    }
}